src/main/java/de/ait/secondlife/exception_handling/GlobalExceptionHandler.java

```diff
@@ -58,10 +58,10 @@ public ResponseEntity<ResponseMessageDto> handleException(NoRightToChangeExcepti
     public ResponseEntity<ResponseMessageDto> handleException(ParameterNotFoundException e) {
         return new ResponseEntity<>(new ResponseMessageDto(e.getMessage()), HttpStatus.NOT_FOUND);
     }
-    @ExceptionHandler(RuntimeException.class)
-    public ResponseEntity<ResponseMessageDto> handleException(RuntimeException e) {
-        return new ResponseEntity<>(new ResponseMessageDto("Something went wrong"), HttpStatus.INTERNAL_SERVER_ERROR);
-    }
+//    @ExceptionHandler(RuntimeException.class)
```

Comment:
this code was commented on temporarily, I guess?

Reply:
fix. and add TODO than some exception take not informative messages